caikit/core/modules/meta.py

```diff
@@ -142,6 +143,9 @@ def metadata_injecting_load(clz, *args, **kwargs):
 
                 return module
 
+            metadata_injecting_load = functools.wraps(real_load.__func__)(
```

Comment:
nit: just because future me might forget whats the intention with `metadata_injecting_load`, can we add a comment here explaining ?

Reply:
Heh, the old "you poked it you own it" comment! I was just adding this wrapping so that the inspected signature is preserved by this wrapper (which is the right thing to do, regardless of _why_ I'm doing it). Is the request that I comment on why I'm using `functools.wraps`, or a more holistic comment on what this whole `metadata_injecting_load` business is about? If it's the later, I think I need that to come from @joerunde

Reply:
Adding a comment about the wrapping, but I'm going to not add anything new about what this wrapper is doing in the first place

Reply:
hehe, it was more for the later .. But make sense ;)